jdenticon/jdenticon-tests.ts

```diff
@@ -0,0 +1,11 @@
+import {jdenticon} from "./index";
```

Comment:
OK, but should I change it to `import {jdenticon} from "jdenticon";` or `import jdenticon from "jdenticon";`?  (The difference being the `{}`.)